### PR TITLE
Update FindRefs to find constructor references in a primary constructor base type

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Fixes https://github.com/dotnet/roslyn/issues/73704

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
@ToddGrun ptal

File: src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/ConstructorSymbolReferenceFinder.cs [hunk 1]
```diff
@@ -257,4 +260,45 @@ private static void FindReferencesInImplicitObjectCreationExpression<TData>(
...
+                continue;
+
+            var constructor = state.SemanticModel.GetSymbolInfo(node, cancellationToken).Symbol;
+            if (Matches(constructor, symbol))
+            {
+                var result = new FinderLocation(node, new ReferenceLocation(
+                    state.Document,
+                    alias: null,
+                    token.GetLocation(),
+                    isImplicit: true,
```
Comment (ToddGrun) [inline src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/ConstructorSymbolReferenceFinder.cs:297]: >isImplicit: true, [](http://example.com/codeflow?start=20&length=17)

just want to check that isImplicit should be true here

Comment (CyrusNajmabadi) [inline src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/ConstructorSymbolReferenceFinder.cs:297]:
```suggestion
                    isImplicit: false,
```

**Review by ToddGrun — APPROVED** [review]
:shipit:

**Pull request merged**